tests/b028.py

```diff
@@ -11,3 +11,7 @@
 warnings.warn("test", DeprecationWarning, stacklevel=1)
 warnings.warn("test", DeprecationWarning, 1)
 warnings.warn("test", category=DeprecationWarning, stacklevel=1)
+args = ("test", DeprecationWarning, 1)
+warnings.warn(*args)
+kwargs = {"message": "test", "category": DeprecationWarning, "stacklevel": 1}
+warnings.warn(**kwargs)
```

Comment:
```suggestion
warnings.warn(*args)
kwargs = {"message": "test", "category": DeprecationWarning, "stacklevel": 1}
warnings.warn(**kwargs)
warnings.warn(*args, **kwargs)
```
Do we need to handle both? If so, let's explicitly test.